src/load/azext_load/data_plane/help.py

```diff
@@ -84,3 +84,19 @@
 short-summary: Command group to retrieve load test run metrics.
 long-summary: Command group to retrieve load test run metrics with list, get-namespaces, get-definitions, get-dimension.
 """ + _common_params
+
+helps[
+    "load trigger"
+] = """
+type: group
+short-summary: Command group to manage load trigger.
```

Comment:
"Command to manage triggers"?
Nikita can also chime in

Same below

Reply:
Command group to manage triggers